images/prow-tests/Dockerfile

```diff
@@ -30,6 +30,7 @@ RUN apt-get install -y uuid-runtime  # for uuidgen
 RUN apt-get install -y npm  # for markdown-link-check
 RUN apt-get install -y rubygems  # for mdl
 RUN apt-get install -y build-essential libssl-dev # for wrk
+RUN apt-get install -y netbase # sets up /etc/services needed for network calls
```

Comment:
I would just comment this as "for wrk". There's nothing else network related that needs netbase.

Reply:
done